.kokoro/release/publish_javadoc11.sh

```diff
@@ -27,6 +27,11 @@ fi
 # work from the git root directory
 pushd $(dirname "$0")/../../
 
+python3 --version
+
+# Force latest protobuf version
+python3 -m pip install protobuf
```

Comment:
We should avoid using floating versions.
cc/ @mpeddada1

Reply:
This was just to test our theory about not pulling the latest version of protobuf. Going to remove.